tests/README.md

```diff
@@ -184,6 +184,20 @@ TIMEOUT_COLLECT_TEST_RESULTS=20
 
 These can be set either in the `environment` section of the tmt plan or using the `-e` option when running tmt, e.g. `-eTIMEOUT_TEST_SETUP=40`.
 
+### Test-specific timeouts
+
+In addition to the mentioned above timeouts, there's a mechanism allowing to set values in specific tests via environment variables, which can be used to adjust test-specific timeouts via the environment. The expected environment variable name would be assembled from the prefix `TEST_`, test name and a provided suffix.
+
```

Comment:
I'd add here that these timeouts/env vars can be set in the tmt plan such as container.fmf in the `environment` section.

Reply:
It is described a couple of lines above, in the previous section. Do you think this is still necessary?

Reply:
Ah, didn't know about that. Maybe still worth mentioning it, e.g. like this:
```
These test specific timeouts can be set in the tmt plan as described [here](#changing-timeouts-for-integration-tests). 
```
But up to you. I'll approve anyway :grin:

Reply:
Added :)